test/catalog2.c

```diff
@@ -62,7 +62,6 @@ ODBC_TEST(t_bug37621)
   if (iType != SQL_VARCHAR && iType != SQL_WVARCHAR)
     return FAIL;
   /* This can fail for the same reason as t_bug32864 */
-  is_num(uiDef, 2048);
```

Comment:
I've removed this assertion as it doesn't make much sense and fails with the newest SingleStore server.